.github/workflows/delivery-docker.yml

```diff
@@ -33,29 +33,30 @@ jobs:
             }
 
             return tag.replace(/^v/, '');
-      - name: Checkout workflow
-        uses: actions/checkout@v2
-      - name: Checkout source at tag
+      - name: Checkout delivery configuration
         uses: actions/checkout@v2
         with:
           path: ./src
+      - name: Checkout workflow
```

Comment:
Was this not clearer as "Checkout source at tag"?  workflow and delivery config sound to be the same.

Reply:
I reverted that. Is it better to check out the src to `/src` and the configuration to `/`, or to have the source be the home, while the configuration is `/somewhere`?